packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -178,57 +245,168 @@ const startNewEconomicCommittee = async ({
+  economicCommittee.resolve(instance);
 
-  // reset because it's already been resolved
   economicCommitteeCreatorFacet.reset();
   economicCommitteeCreatorFacet.resolve(creatorFacet);
-  return newPoserInvitationP;
 };
 harden(startNewEconomicCommittee);
 
-const main = async permittedPowers => {
-  console.log('starting new economic committee:', runConfig);
-  const newElectoratePoser = await startNewEconomicCommittee(permittedPowers);
+const startNewEconCharter = async ({
+  consume: { zoe },
+  produce: { econCharterKit },
+  installation: {
+    consume: { binaryVoteCounter: counterP, econCommitteeCharter: installP },
+  },
+  instance: {
+    produce: { econCommitteeCharter: instanceP },
+  },
+}) => {
+  const [charterInstall, counterInstall] = await Promise.all([
+    installP,
+    counterP,
+  ]);
+  const terms = await harden({
+    binaryVoteCounterInstallation: counterInstall,
+  });
 
-  /*
-   * put the new economic committee into agoricNames
-   */
+  trace('Starting new EC Charter Instance');
+  const startResult = await E(zoe).startInstance(
+    charterInstall,
+    undefined,
+    terms,
+    undefined,
+    'econCommitteeCharter',
+  );
+  trace('Started new EC Charter Instance Successfully');
+
+  instanceP.reset();
+  instanceP.resolve(E.get(startResult).instance);
+
+  econCharterKit.reset();
+  econCharterKit.resolve(startResult);
+};
+harden(startNewEconCharter);
+
+const addGovernorsToEconCharter = async ({
+  consume: {
+    reserveKit,
+    vaultFactoryKit,
+    econCharterKit,
+    auctioneerKit,
+    psmKit,
+  },
+  instance: {
+    consume: { reserve, VaultFactory, auctioneer },
+  },
+}) => {
+  const { creatorFacet } = E.get(econCharterKit);
+
+  const psmKitMap = await psmKit;
+
+  for (const obj of psmKitMap.values()) {
+    await E(creatorFacet).addInstance(
+      obj.psm,
+      obj.psmGovernorCreatorFacet,
+      obj.label,
+    );
+  }
+
+  const instances = [
+    {
+      label: 'reserve',
+      instanceP: reserve,
+      facetP: E.get(reserveKit).governorCreatorFacet,
+    },
+    {
+      label: 'VaultFactory',
+      instanceP: VaultFactory,
+      facetP: E.get(vaultFactoryKit).governorCreatorFacet,
+    },
+    {
+      label: 'auctioneer',
+      instanceP: auctioneer,
+      facetP: E.get(auctioneerKit).governorCreatorFacet,
+    },
+  ];
+
+  for (const { label, instanceP, facetP } of instances) {
+    const instance = await instanceP;
+    const govFacet = await facetP;
+    await E(creatorFacet).addInstance(instance, govFacet, label);
+  }
+};
+
+harden(addGovernorsToEconCharter);
+
+const shutdown = async ({
+  consume: { economicCommitteeCreatorFacet, econCharterKit },
+}) => {
+  console.log('Shutting down old EC Committee');
+  await E(economicCommitteeCreatorFacet).shutdown();
+  console.log('EC Committee shutdown successful');
+
+  console.log('Shutting down old EC Charter');
+  const { creatorFacet } = E.get(econCharterKit);
+  await E(creatorFacet).shutdown();
```

Comment:
This method is available on the new committee, but not the old one, unless the old one got upgraded.